proto/axelar/batch/v1beta1/tx.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package axelar.batch.v1beta1;
+
+import "google/protobuf/any.proto";
+import "gogoproto/gogo.proto";
+
+import "axelar/permission/exported/v1beta1/types.proto";
+import "cosmos/base/abci/v1beta1/abci.proto";
+
+option go_package = "github.com/axelarnetwork/axelar-core/x/batch/types";
+
+message BatchRequest {
+  option (permission.exported.v1beta1.permission_role) = ROLE_UNRESTRICTED;
+
+  bytes sender = 1 [ (gogoproto.casttype) =
+                         "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
+  repeated google.protobuf.Any messages = 2
+      [ (gogoproto.nullable) = false ];
+}
+
+message BatchResponse {
+  repeated cosmos.base.abci.v1beta1.Result  results = 1;
+}
```

Comment:
```suggestion
message BatchResponse {
  message Response {
    oneof res {
      cosmos.base.abci.v1beta1.Result result = 1;
      string err = 2;
    }
  }

  repeated Response responses = 1;
}
```
I dislike the current definition for the fact that failed messages would have a nil result.